src/examples/color-picker/ColorpickerDialogCustomTrigger.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { ColorpickerDialog, IColor } from '@zendeskgarden/react-colorpickers';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Tooltip } from '@zendeskgarden/react-tooltips';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { ReactComponent as PaletteIcon } from '@zendeskgarden/svg-icons/src/16/palette-fill.svg';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+interface IPaletteIconButton {
+  iconColor: string;
+}
+
+const PaletteIconButton = React.forwardRef(
+  (
+    props: IPaletteIconButton & React.ComponentPropsWithoutRef<'button'>,
+    ref: React.Ref<HTMLButtonElement>
+  ) => (
+    <Tooltip content="Palette">
+      <IconButton aria-label="palette" ref={ref} {...props}>
+        <PaletteIcon style={{ color: props.iconColor }} />
```

Comment:
The alpha value doesn't appear to be getting applied to this icon.

Reply:
Looks like the alpha changes broke using the hex color from `DEFAULT_THEME`.  The value is now passed to `rgba` and alpha should now be applied to the icon.